mkdocs.yml

```diff
@@ -64,7 +64,7 @@ nav:
   - 'Setup': setup.md
   - 'Add Dependencies': add-dependencies.md
   - 'Update Dependencies': update-dependencies.md
-#  - 'Gradle buildSrcVersions': gradle-buildsrcversions.md
+  - 'Gradle buildSrcVersions': gradle-buildsrcversions.md
```

Comment:
This change is unrelated to this PR (out of the scope), plus we previously agreed that we would not make it part of the main menu for now.